Update day1_2_source_degradation

PheeYongHan opened the pull request:
Updated with py-pde 0.44.0

PheeYongHan committed:
Update day1_2_source_degradation_solution.ipynb

Updated with py-pde 0.44.0

PheeYongHan committed:
Update day1_2_source_degradation.ipynb

Updated with py-pde 0.44.0

david-zwicker commented:
Thanks for starting this! I suspect there will be more incompatibilities in the other notebooks, so it might be a good idea to correct all of them in one pull request. I will also try to add a script that tests automatically whether all Jupyter notebooks run through!

PheeYongHan committed:
Update day1_3_reaction-diffusion_solution.ipynb

PheeYongHan committed:
Update day1_3_reaction-diffusion.ipynb

PheeYongHan committed:
Merge branch 'zwicker-group:main' into main

PheeYongHan committed:
Update day2_1_pole-to-pole.ipynb

PheeYongHan committed:
Update day2_1_pole-to-pole_solution.ipynb

PheeYongHan committed:
Merge branch 'main' of https://github.com/PheeYongHan/tutorial-pattern-formation-in-cells

PheeYongHan committed:
Update day2_2_physics-of-diffusion_solution.ipynb

PheeYongHan committed:
Update day2_2_physics-of-diffusion.ipynb

PheeYongHan committed:
Revert "Update day2_2_physics-of-diffusion_solution.ipynb"

This reverts commit 027b8a62b55629b77b58d9076cc181a1b2e69458.

PheeYongHan committed:
Display gitignore

PheeYongHan committed:
Update day2_3_non-ideal-reaction-diffusion.ipynb

PheeYongHan commented:
I have gone through the remaining notebooks and updated them accordingly. Hopefully it all works well now!

david-zwicker commented:
Great! Thank you very much! I'll have a look at everything when I'm back from holidays.

david-zwicker committed:
Fix comments in tutorial notebooks and update test compatibility script

- Updated the comment in day1_2_source_degradation.ipynb, day1_3_reaction-diffusion.ipynb, and day2_1_pole-to-pole.ipynb to correct the function name from "_make_derivative2" to "make_derivative2".
- Increased the timeout for nbmake tests in test_compatibility.sh to 1000 seconds to prevent timeouts during execution.
